DNN Platform/Library/Framework/CDefault.cs

```diff
@@ -121,7 +121,7 @@ protected string AdvancedSettingsPageUrl
                 if (modules.Count > 0)
                 {
                     var pmb = new PortalModuleBase();
-                    result = pmb.EditUrl(tab.TabID, "", false, "mid=" + modules.ElementAt(0).ModuleID, "popUp=true", "ReturnUrl=" + Server.UrlEncode(TestableGlobals.Instance.NavigateURL()));
+                    result = pmb.EditUrl(tab.TabID, "", false, string.Concat("mid=", modules.ElementAt(0).ModuleID), "popUp=true", string.Concat("ReturnUrl=", Server.UrlEncode(TestableGlobals.Instance.NavigateURL())));

```

Comment:
The `+` operator compiles to `string.Concat`, so there is no different between the two (see http://stackoverflow.com/q/1787696/2688).  At some point it make sense to switch from concatenating to using a `StringBuilder`, but changing from `+` to `Concat` has no difference in the underlying code that is generated (except in the case where you have two constant expression, in which case the compiler will pre-concatenate them when using `+`, before transforming it into a `Concat` call)